spec/content/automate/ManageIQ/Cloud/Orchestration/Provisioning/StateMachines/Methods.class/__methods__/check_provisioned_spec.rb

```diff
@@ -110,6 +110,16 @@
       expect(ae_service.root['ae_result']).to eq("retry")
     end
 
+    it "does not need to wait for refresh completes if the stack has been removed from provider" do
+      ae_service.set_state_var('provider_last_refresh', true)
+      ae_service.set_state_var('deploy_result', 'error')
+      allow(svc_model_service)
+        .to receive(:orchestration_stack_status) { ['check_status_failed', 'stack does not exist'] }
+      amazon_stack.update_attribute(:ems_ref, nil)
```

Comment:
@bzwei Per robocop warning would be better to use `amazon_stack.update_attributes(:ems_ref => nil)`